Doc/howto/regex.rst

```diff
@@ -468,9 +491,9 @@ Two pattern methods return all of the matches for a pattern.
    ['12', '11', '10']
 
 The ``r`` prefix, making the literal a raw string literal, is needed in this
-example because escape sequences in a normal "cooked" string literal that are
-not recognized by Python, as opposed to regular expressions, now result in a
-:exc:`DeprecationWarning` and will eventually become a :exc:`SyntaxError`.  See
+example because ``\d`` is not an escape sequence recognized in Python string literals.
+Such unrecognized sequences now produce a
+:exc:`SyntaxWarning` and will eventually become a :exc:`SyntaxError`.  See
 :ref:`the-backslash-plague`.
```

Comment:
This feels redundant.
```suggestion
See :ref:`the-backslash-plague`.
```

Reply:
Applied; I decided to also remove the link.